synapse/lib/stormtypes.py

```diff
@@ -289,15 +293,36 @@ def __init__(self, node, path=None):
             'repr': self._methNodeRepr,
             'iden': self._methNodeIden,
             'value': self._methNodeValue,
+            'globtags': self._methNodeGlobTags,
         })
 
     async def _methNodeTags(self, glob=None):
         tags = list(self.valu.tags.keys())
+        tags.sort()
```

Comment:
Any particular reason we're sorting these?

Reply:
this ensures that, for a node which was freshly constructed vs. a node with tags lifted out of a storage backing, we'd get consistent results.  Removing the sorts allows the current unit tests to pass by luck; reordering the tags causes them to fail.  Can change the tests to just check for sets of results.